### DISCUSSION TITLE
笔记树的文件排序问题

### DISCUSSION THREAD

**Discussion by dbkuaizi** [tamlok/vnote#1948] [General]
选择按名称排序，但并没有和 typora 一样 按照预期排序
![Snipaste_2021-11-28_12-53-24](https://user-images.githubusercontent.com/44014601/143730123-86fddd59-2b91-4f10-8e7c-a9d77b0470cd.jpg)

**Comment by tamlok** [comment]
从计算机意义上来说，vnote的才是字符串排序，你可以写成01/02/03。

**Reply by dbkuaizi** [reply]
好的 了解了

**Reply by jovensun** [reply]
右边使用的应该是 NaturalComparer 或者 NaturalSort。
wiki: [Natural sort order](https://en.wikipedia.org/wiki/Natural_sort_order)
